es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 
 <br>
 
 **4. Confusion matrix ― The confusion matrix is used to have a more complete picture when assessing the performance of a model. It is defined as follows:**
 
-&#10230;
+&#10230; Matriz de confusión - la matriz de confusión se utiliza para tener una visión más completa al evaluar el rendimiento de un modelo. Se define de la siguiente manera:
 
 <br>
 
 **5. [Predicted class, Actual class]**
 
-&#10230;
+&#10230; [Clase predicha, clase real]
 
 <br>
 
 **6. Main metrics ― The following metrics are commonly used to assess the performance of classification models:**
 
-&#10230;
+&#10230; Métricas principales - las siguientes métricas se utilizan comúnmente para evaluar el rendimiento de los modelos de clasificación:
 
 <br>
 
 **7. [Metric, Formula, Interpretation]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Interpretación]
 
 <br>
 
 **8. Overall performance of model**
 
-&#10230;
+&#10230; Rendimiento general del modelo
 
 <br>
 
 **9. How accurate the positive predictions are**
 
-&#10230;
+&#10230; Que tan precisas son las predicciones positivas
 
 <br>
 
 **10. Coverage of actual positive sample**
 
-&#10230;
+&#10230; Cobertura de la muestra positiva real
 
 <br>
 
 **11. Coverage of actual negative sample**
 
-&#10230;
+&#10230; Cobertura de la muestra negativa real
 
 <br>
 
 **12. Hybrid metric useful for unbalanced classes**
 
-&#10230;
+&#10230; Métrica híbrida útil para clases desbalanceadas
 
 <br>
 
 **13. ROC ― The receiver operating curve, also noted ROC, is the plot of TPR versus FPR by varying the threshold. These metrics are are summed up in the table below:**
 
-&#10230;
+&#10230; ROC ― La curva Característica Operativa del Receptor, también conocida como ROC, es una representación gráfica de la sensibilidad frente a la especificidad según se varía el umbral. Estas métricas se resumen en la tabla a continuación:
 
 <br>
 
 **14. [Metric, Formula, Equivalent]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Interpretación]
 
 <br>
 
 **15. AUC ― The area under the receiving operating curve, also noted AUC or AUROC, is the area below the ROC as shown in the following figure:**
 
-&#10230;
+&#10230; AUC ― El área bajo la curva Característica Operativa del Receptor, también conocida como AUC o AUROC, es el área debajo del ROC, como se muestra en la siguiente figura:
 
 <br>
 
 **16. [Actual, Predicted]**
 
-&#10230;
+&#10230; [Actual, predicha]
 
 <br>
 
 **17. Basic metrics ― Given a regression model f, the following metrics are commonly used to assess the performance of the model:**
 
-&#10230;
+&#10230; Métricas básicas - dado un modelo de regresión f, las siguientes métricas se usan comúnmente para evaluar el rendimiento del modelo:
 
 <br>
 
 **18. [Total sum of squares, Explained sum of squares, Residual sum of squares]**
 
-&#10230;
+&#10230; [Suma total de cuadrados, suma de cuadrados explicada, suma residual de cuadrados]
```

Comment:
[Suma total de cuadrados, Suma de cuadrados explicada, Suma residual de cuadrados]

Only change to capital letters, so that it is similar to the English version